docs/05-migration/01-about.md

```diff
@@ -0,0 +1,82 @@
+# Migration
+
+Sometimes also described as reindexing or reingestion.
+
+Image migration is an optional process in the Grid that will create a new index
+in the Elasticsearch cluster containing an up-to-date representation of all
+known images. In simple cases, this is akin to the [Elasticsearch Reindex
+API][es-reindex], i.e. moving all documents to a new index using the latest
+version of the mapping, but migration also contains improvements to ensure a
+seamless transition and improved metadata.
```

Comment:
I would be hesitate to liken it to reindex API, I think the key thing to convey is that it's rebuilding the ES index from the original image metadata (in S3) AND the user metadata (in DynamoDB) and NOT from the existing ES representation.
```suggestion
known images. Unlike the [Elasticsearch Reindex
API][es-reindex] (i.e. copying all documents to a new index using the latest
version of the mapping) the migration process rebuilds the document  from the original image metadata (in S3) AND the user metadata (in DynamoDB) and NOT from the existing ES representation. 
Migration also contains improvements to ensure a seamless transition and improved metadata.
```

Reply:
I concur: great work, thank you!

Regarding similarities/differences to reindexing, I think this needs to die (or be adjusted):
https://github.com/guardian/grid/blob/main/docs/05-reingestion/01-how-to.md

Similarly, not sure how much of this still applies or how it should change because of migration?
https://github.com/guardian/grid/tree/main/scripts#reindex
&
https://github.com/guardian/grid/tree/main/scripts#updatemapping

Reply:
Thanks (sorry for not spotting binning!)